src/main/java/org/apache/mesos/dcos/Capabilities.java

```diff
@@ -0,0 +1,19 @@
+package org.apache.mesos.dcos;
+
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+/**
+ * This class represents a set of capabilities that may or may not be supported in a given version of DC/OS.
+ */
+public class Capabilities {
+    private DcosCluster dcosCluster;
+
+    public Capabilities(DcosCluster dcosCluster) {
```

Comment:
If DcosCluster URI can be autodetected (see comment there): Construct DcosCluster internally?
Else: Accept URI directly and construct DcosCluster from that?